evap/rewards/tests/test_views.py

```diff
@@ -80,10 +80,20 @@ def test_redeem_points_for_expired_event(self):
         self.assertEqual(5, reward_points_of_user(self.student))
 
     def test_invalid_post_parameters(self):
-        self.app.post(self.url, params={"points-asd": 2}, user=self.student, status=400)
-        self.app.post(self.url, params={"points-": 2}, user=self.student, status=400)
-        self.app.post(self.url, params={f"points-{self.event1.pk}": ""}, user=self.student, status=400)
-        self.app.post(self.url, params={f"points-{self.event1.pk}": "asd"}, user=self.student, status=400)
+        point_params = {
+            "previous_reward_points": reward_points_of_user(self.student),
+            "previous_redeemed_points": redeemed_points_of_user(self.student),
+        }
+        self.app.post(self.url, params={"points-asd": 2, **point_params}, user=self.student, status=400)
```

Comment:
This is now very lengthy, you could add a helper:
```python
def helper(redemption_params, additional_params=correct_previous_params, status=400):
    self.app.post(self.url, params={**redemption_params, **additional_params}, user=self.student, status=status)
```
You can then refactor all `post` calls to use the helper. I would also suggest not mutating `point_params` below and instead just making a new `incorrect_previous_params`